rpcs3/Emu/Cell/Modules/cellSsl.cpp

```diff
@@ -52,11 +60,15 @@ s32 cellSslCertificateLoader(u64 flag, vm::ptr<char> buffer, u32 size, vm::ptr<u
 	if (required)
 	{
 		*required = (u32)flagBits.count();
-		for (int i = 1; i < flagBits.size(); i++)
+		for (int i = 1; i <= flagBits.size(); i++)
```

Comment:
Why it's `<=` here while it's `<` in the `else` branch?

Reply:
woops, thanks, good catch.